.drone.yml

```diff
@@ -7,7 +7,7 @@ platform:
 
 steps:
 - name: ci
-  image: rancher/hardened-build-base:v1.15.8b5
+  image: golang:1.16
```

Comment:
I think before we actually ship we should unify everything on dapper. But for now this is fine.